spec/services/user_migration_service_spec.rb

```diff
@@ -3,25 +3,19 @@
 require "rails_helper"
 
 RSpec.describe UserMigrationService do
-  let(:user_migration_service) do
-    UserMigrationService.new
+  subject do
+    UserMigrationService.run
```

Comment:
I am not sure I like using `subject` here, one for discussion :D

Reply:
@cintamani Swapped it for the format we used on the other services – how's this? https://github.com/DEFRA/waste-carriers-back-office/pull/439/commits/b8e18795f17049cfcbfde094472bae36c14ccf59

Reply:
Nice!! What if we call it `result` rather than `service`? IMO seems clearer what the variable holds with `result`

Reply:
My NIT for this is I don't think we should bother with this let at all. That might be because I don't understand how it's helping.

It would be clearer to me if I just saw `UserMigrationService.run` instead of `service` in the tests, or `described_class.run` to avoid lots of changes should we rename it.

But as I said, there might be a benefit or convention of putting the execution in a `let` I'm just not aware of.

Reply:
I think the benefit of having it in a `let` is that if the execution of the service changes we have to update 1 line of code only

Reply:
The other PR helped me see the light https://github.com/DEFRA/waste-carriers-back-office/pull/440#pullrequestreview-316107268 !

Reply:
Perhaps `run_service` 🤷‍♂